views/js/e2e/runner/_routes/runnerRoutes.js

```diff
@@ -34,4 +34,8 @@ Cypress.Commands.add('addRoutes', () => {
     cy.route('POST', '**/delete').as('delete');
     cy.route('POST', '**/deleteClass').as('deleteClass');
     cy.route('POST', '/tao/GenerisTree/getData').as('getData');
-});
\ No newline at end of file
+
+    // Register routes for delivery execution
+    cy.route('POST', '/taoQtiTest/Runner/**').as('testRunnerPost'); // Should be done with a regex -> '/\/taoQtiTest\/Runner\/init\S+/'
+    cy.route('GET', '/taoQtiTest/Runner/**').as('testRunnerGet'); // Should be done with a regex -> '/\/taoQtiTest\/Runner\/getItem\S+/'
+});
```

Comment:
Now it's ok, but maybe after a while, we have to consider to separate the routes to files because maybe not all the routes have to be registered or added to all thee test cases.

Reply:
Yes, you're right. navigation tests only need these 2. I will separate them into a `testExecutionRoutes` file.